src/PresentationalComponents/ImpactBadge/ImpactBadge.js

```diff
@@ -0,0 +1,85 @@
+import { QuestionIcon, SecurityIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React from 'react';
+
+class ImpactBadge extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.colorList = {
+            default: 'var(--pf-global--Color--200)', // grey
+            danger: 'var(--pf-global--danger-color--100)',
+            warning: 'var(--pf-global--warning-color--100)',
+            orange: '#ec7a08' // orange
+        };
+
+        /* I'm not shure that the list of impacts is ordered correctly by relevancy */
+        this.impactList = {
+            Critical: {
```

Comment:
Can we safely assume these values will always be uppercase?

Reply:
cc @jiridostal Please can you answer and make a conclusion?

Reply:
As Vulnerability engine stores impacts as a dictionary of uppercase values, we can assume it will either be uppercase word or empty string.